lang/funcs/engine.go

```diff
@@ -450,10 +450,6 @@ func (obj *Engine) Run() error {
 				// XXX: maybe we can get rid of the table...
 				obj.table[vertex] = value // save the latest
 				node.mutex.Lock()
-				if err := node.Expr.SetValue(value); err != nil {
```

Comment:
This is the call to `SetValue()` which seems redundant to me, because the engine is already storing the latest value in `obj.table[vertex]` two lines above.